src/Psy/VarDumper/Presenter.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of Psy Shell
+ *
+ * (c) 2012-2014 Justin Hileman
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Psy\VarDumper;
+
+use Symfony\Component\Console\Formatter\OutputFormatter;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\VarDumper\Caster\Caster;
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * A Presenter service.
+ */
+class Presenter
+{
+    const VERBOSE = 1;
+
+    private $cloner;
+    private $dumper;
+    private $exceptionsImportants = array(
+        "\0*\0message",
+        "\0*\0code",
+        "\0*\0file",
+        "\0*\0line",
+        "\0Exception\0previous",
+    );
+    private $styles = array(
+        'num' => 'number',
+        'const' => 'const',
+        'str' => 'string',
+        'cchr' => 'string',
+        'note' => 'class',
+        'ref' => 'aside',
```

Comment:
I think this one was better the default way… as an `<aside>` it's a bit too bright.